docker/Dockerfile

```diff
@@ -1,23 +1,24 @@
-FROM ubuntu:xenial
+FROM alpine:3.6
```

Comment:
Great, just this has reduced the size of the image from 816 to 516 MB 😄

Reply:
And reduced the download size by a great amount 😌